unlock-app/src/hooks/useAppStorage.ts

```diff
@@ -40,27 +40,23 @@ export function useAppStorage() {
     return null
   }, [])
 
-  const clearStorageWithoutFavorite = useCallback((): void => {
+  /*
+   * @param exclude: string[] - Array of keys to exclude from deletion, only for keys with APP_NAME
+   */
+  const clearStorage = useCallback((exclude?: string[]): void => {
```

Comment:
No! Now that means every piece of code that calls `cleaStorage` needs to know of _all_ objects stored in localstorage otherwise it will erase them.
Please do what I have been recommending for the last 2 days: only have *explicit* deletes, either thru `clearStorage` or thru `removeKey`.